lib/fastlane/plugin/sentry/actions/sentry_create_deploy.rb

```diff
@@ -0,0 +1,103 @@
+module Fastlane
+  module Actions
+    class SentryCreateDeployAction < Action
+      def self.run(params)
+        require 'shellwords'
+
+        Helper::SentryHelper.check_sentry_cli!
+        Helper::SentryConfig.parse_api_params(params)
+
+        version = params[:version]
+        version = "#{params[:app_identifier]}@#{params[:version]}" if params[:app_identifier]
+
+        has_time = !params[:time].nil?
+        has_started = !params[:started].nil?
+        has_finished = !params[:finished].nil?
+
+        if !has_time && (has_started && !has_finished || !has_started && has_finished)
+          UI.user_error!("Only one parmeter of 'started' and 'finished' found, please provide both.")
+        end
```

Comment:
`m`: I'm not sure if we need that. I would just pass all the parameters to sentry-cli and let it handle it. The logic there is a bit different from ours right now and I think it should be the same, see https://github.com/getsentry/sentry-cli/blob/d309c5a607a310eebd23c1a2e49624e6ea192e21/src/commands/releases.rs#L1108-L1121